main.py

```diff
@@ -14,7 +14,9 @@
 
     # start arbitrage detection
     print("Scanning...")
-    exchange_name = "binance"
+    exchanges = ['binance', 'coinbase', 'kucoin', 'binanceus']
+    exchange_name = exchanges[3] # allow pickable exchange_id from https://github.com/ccxt/ccxt/wiki/manual#exchanges
```

Comment:
Could you explain the purpose of the exchanges list? Why not only reference the exchange_d URL instead?

Reply:
I was just testing different exchanges and didn't want to rewrite the entire exchange_id every time... I will get rid of this.